sdks/ios/appbuilder/appbuilder.cs

```diff
@@ -107,7 +107,7 @@ public class AppBuilder
 			aot_args = "soft-debug";
 		if (isllvm) {
 			cross_runtime_args = "--llvm";
-			aot_args = ",llvm-path=$mono_sdkdir/ios-llvm64/bin,llvm-outfile=$llvm_outfile";
+			aot_args = ",llvm-path=$mono_sdkdir/../out/llvm-llvm64/bin,llvm-outfile=$llvm_outfile";
 		}
```

Comment:
Xamarin.ios expects to find llvm in the previous directory.

Reply:
I have no opinions here. @luhenry moved the directory if I'm not mistaken. I can move it back if the plan isn't to move it, or I can keep it.

Reply:
The directory name is part of the api between the sdk and the products, it should not be changed unless really neccesarry.

Reply:
What else other than cross-compilers depends on the location of this LLVM?

Reply:
The xamarin.ios install scripts.

Reply:
Ok, let's revert that change then.